lib/core/device.js

```diff
@@ -274,7 +271,39 @@ device.detectUsb = function(driverId, cb) {
       device.othersConnected = devices.length - 1;
     }
     return cb(null, devdata);
-  });
+  };
+
+  if (self._os === 'win') {
+    var chromeVersion = /Chrome\/([0-9]+)/.exec(navigator.userAgent)[1];
+    self.log('Chrome version detected as', chromeVersion);
+
+    chrome.serial.getDevices(function(results) {
+      if (chromeVersion >= 50) {
+        // As of Chrome 50, product ID and vendor ID is returned
```

Comment:
In comment, can you link to the chrome/chromium issue that shows the bug or fix?